### DISCUSSION TITLE
Allow a pack to not be affected by a roles discounts.

### DISCUSSION THREAD

**Discussion by PoeBloodstone** [gameserverapp/Platform#510] [Suggestions]
Allow a pack to not be affected by a roles discounts at all, Example if a person has a VIP role that has a 50% discount and there is a pack to turn tokens into the token gem in-game, the person can then buy 20 gems for 10 tokens, then consume and recover the 20, then buy another 20 for 10 then keep on doing it for infinite currency. We currently can't do in-game trading due to this explot from our supporter members.

**Comment by mvaessen** [comment]
duplicate of #494
